src/Gui/ViewProviderGeometryObject.cpp

```diff
@@ -115,44 +130,32 @@ ViewProviderGeometryObject::~ViewProviderGeometryObject()
 
 void ViewProviderGeometryObject::onChanged(const App::Property* prop)
 {
-    // Actually, the properties 'ShapeColor' and 'Transparency' are part of the property 'ShapeMaterial'.
-    // Both redundant properties are kept due to more convenience for the user. But we must keep the values
-    // consistent of all these properties.
+    // Actually, the properties 'ShapeColor' and 'Transparency' are part of the property
+    // 'ShapeMaterial'. Both redundant properties are kept due to more convenience for the user. But
+    // we must keep the values consistent of all these properties.

```

Comment:
This comment probably doesn't make sense anymore, since onChanged doesn't have a conditional for ShapeColor.